.eslintrc.js

```diff
@@ -7,54 +7,54 @@ module.exports = {
     ecmaVersion: 2018,
     sourceType: 'module',
     ecmaFeatures: {
-      legacyDecorators: true
-    }
+      legacyDecorators: true,
+    },
   },
-  "globals": {
-    "flatpickr": false
+  globals: {
+    flatpickr: false,
   },
-  plugins: [
-    'ember'
-  ],
+  plugins: ['ember'],
   extends: [
     'eslint:recommended',
-    'plugin:ember/recommended'
+    'plugin:ember/recommended',
+    'plugin:prettier/recommended',
   ],
   env: {
-    browser: true
+    browser: true,
   },
   rules: {
-    'ember/no-jquery': 'error',
-    "ember/no-mixins": "warn",
-    "semi": "error",
+    'ember/avoid-leaking-state-in-ember-objects': 'warn',
+    'ember/no-classic-classes': 'warn',
```

Comment:
Both of these rules can be removed once everything has been converted to native classes. I'll create a separate PR for that.